packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -703,10 +703,13 @@ export class ContainerRuntime
 			context,
 			registryEntries,
 			existing: existingFlag,
-			requestHandler,
 			runtimeOptions,
 			containerScope,
 			containerRuntimeCtor,
+			requestHandler,
+			initializeEntryPoint: () => {
+				throw new UsageError("initializeEntryPoint was not provided");
```

Comment:
nit: The message can be something that says this was called via load which is deprecated and shouldn't be used.